base/archive/zip_test.go

```diff
@@ -92,9 +92,13 @@ func TestWriteZipFullDataset(t *testing.T) {
 		return
 	}
 
-	tmppath := filepath.Join(os.TempDir(), "exported.zip")
-	// defer os.RemoveAll(tmppath)
-	t.Log(tmppath)
+	path, err := ioutil.TempDir("", "TestArchiveFullDataset")
+	if err != nil {
+		t.Fatal(err)
+	}
+	tmppath := filepath.Join(path, "exported.zip")
+	defer os.RemoveAll(tmppath)
+	// t.Log(tmppath)
```

Comment:
delete dead code

Reply:
yar, this comment is here as an indication on how to un-break the test when the golden file needs updating. I've added a clarifying comment